pkg/controller/persistentvolume/persistentvolume_provisioner_controller.go

```diff
@@ -345,6 +347,11 @@ func newProvisioner(plugin volume.ProvisionableVolumePlugin, claim *api.Persiste
 		AccessModes:                   claim.Spec.AccessModes,
 		PersistentVolumeReclaimPolicy: api.PersistentVolumeReclaimDelete,
 		CloudTags:                     &tags,
+		ClusterName:                   controller.clusterName,
+	}
+
+	if pv != nil {
+		volumeOptions.PVName = pv.Name
```

Comment:
A comment would help.

It is guaranteed unique because we first persist the PV with, for example, `GenerateName: "pv-aws-", objectMeta.GenerateName`

We then want to use that PV name as the resource name to tie them together.